corehq/apps/locations/models.py

```diff
@@ -805,6 +813,24 @@ def _unassign_users_from_location(domain, location_id):
             user.unset_location_by_id(location_id, fall_back_to_next=True)
 
 
+def get_case_sharing_groups_for_locations(cls, locations, for_user_id=None):
```

Comment:
Looks like you forgot to finish converting this from a class method
```suggestion
def get_case_sharing_groups_for_locations(locations, for_user_id=None):
```

Reply:
Haha, I did. Sorry, fixed it now.